### ISSUE TITLE
Can't manage to make multiChart work

### ISSUE THREAD

**Issue by jpagand** [krispo/angular-nvd3#154]
Starting from this example 
http://plnkr.co/edit/zsNxBJ?p=preview

Within a directive, I try to build a multiChart with just a area chart for now

```javascript
        scope.nvOptions = {
          chart: {
            type: 'multiChart',
            height: 400,
            margin : {
              top: 10,
              bottom: 10
            },
            color: function () { return '#322d44'; },
            useInteractiveGuideline: true,
            transitionDuration: 500,
            yAxis1: {
              tickFormat: function(d){
                return d3.format(',.1f')(d);
              }
            },
            xAxis: {
              tickFormat: function(d){
                return d3.format(',f')(d);
              }
            }            
          }
        };

        $http.get('/api/activity')
              .success(function (data) {
                var values = [];
                angular.forEach(data, function (d) {
                 values.push({x: d.hour + 2, y: d.count / 3600})
                });
                scope.data = [{
                  key: 'Stream1',
                  type: 'area',
                  yAxis: 1,
                  values: values
                }];
              });
```

```html
<nvd3 options="nvOptions" data="data"></nvd3>
```

Then I got a tons of errors like:

![45dfgh5dhj](https://cloud.githubusercontent.com/assets/2401332/8251044/5ece26f4-1679-11e5-84c8-0c75615e35da.png)

What did I miss?

Thank you!

**Comment by pasine** [comment]
I have the same problem.
I had an issue with my code not working, so I tried copy&paste the example in the plunker but it's not working either.
All the lines and the bars are unresponsive (no hover effect) and whenever I click on the legend labels, the chart is unable to hide/show the different charts in the correct way.
I installed the module through bower, so it should have all the correct dependencies.
The only thing different is the angular version (I am using 1.4.3), but trying to replace it in the original example on plunker gives no issues.
I also tried to use the rc2 version of angular-nvd3, but I found the same behaviour.
Sorry @jpagand how did you fix this?
